kustomize/base-openshift/shard/service-mesh/service-mesh-control-plane.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+  addons:
```

Comment:
Because this is how it should be installed https://docs.openshift.com/container-platform/4.7/service_mesh/v2x/ossm-create-smcp.html#ossm-control-plane-deploy-cli_ossm-create-smcp